Makefile

```diff
@@ -32,9 +33,8 @@ vet:
 	@$(GO) vet $(pkgs)
 
 build:
-	@echo ">> building binaries"
-	@./build/assets.sh
-	@./build/build.sh
+	@echo ">> building binaries using docker"
+	@docker run -i -v $(PWD):/go/src/github.com/google/cadvisor golang:$(goversion) /bin/sh -c "cd /go/src/github.com/google/cadvisor && go get github.com/tools/godep && ./build/assets.sh && ./build/build.sh"
```

Comment:
I don't think it is. I think you need to mount the $GOPATH/pkg directory for it to work.

With your changes (irrelevant output omitted):
```
[(c262eb7...)] cadvisor: time make build
>> building binaries using docker
 >   cadvisor

real	0m14.201s
user	0m0.066s
sys	0m0.032s

[(c262eb7...)] cadvisor: time make build
>> building binaries using docker
 >   cadvisor

real	0m14.279s
user	0m0.073s
sys	0m0.027s
```

Without your changes:
```
[install] cadvisor: time make build
>> building binaries
 >   cadvisor

real	0m6.658s
user	0m13.909s
sys	0m1.622s

[install] cadvisor: time make build
>> building binaries
 >   cadvisor

real	0m0.781s
user	0m0.343s
sys	0m0.188s
```

Notice how the second build is super fast once everything is cached.